src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -226,8 +290,10 @@ private ScheduledInterviewRequest makeScheduledInterviewRequest(
             .get(scheduledInterview.intervieweeId())
             .map(Person::firstName)
             .orElse("Nonexistent User");
-    String shadow =
-        personDao.get(scheduledInterview.shadowId()).map(Person::firstName).orElse("None");
+    String shadow = "None";
+    if (!scheduledInterview.shadowId().equals("")) {
```

Comment:
Some docu explaining why this check is here would be nice.

Reply:
Done